web/annotation_layer_builder.css

```diff
@@ -93,6 +93,64 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation label,
+.annotationLayer .radioButtonWidgetAnnotation label {
+  background-color: rgb(222, 229, 255);
```

Comment:
Here is a sample pdf: [pptc153filled.pdf](https://github.com/mozilla/pdf.js/files/491352/pptc153filled.pdf)